src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -541,6 +541,11 @@ public static bool IsInterpolation(this SyntaxToken currentToken)
             return currentToken.Parent.IsKind(SyntaxKind.Interpolation);
         }
 
+        public static bool IsPattern(this SyntaxToken currentToken)
```

Comment:
it's not clear what this does, token itselfi s not a pattern. Please use a better name a possibly also comment with an example